Buddies/CreateActivity/CreateActivityVC.swift

```diff
@@ -58,6 +59,8 @@ class CreateActivityVC: UITableViewController, UITextViewDelegate, UITextFieldDe
                 description: description,
                 location: GeoPoint(latitude: chosenLocation.latitude,
                                    longitude: chosenLocation.longitude),
+                startTime: getSliderDate(sliderValue: dateSlider.minValue),
```

Comment:
Could you use `start_time` and `end_time` instead? That's what we have in the [data model doc](https://docs.google.com/document/d/1GhrXckvtJGYOKy8wnEAVBvP1aOvDOrLmKcD9wpfECtg/edit) ;)

Reply:
Changed!

Reply:
Oops I'm sorry! I thought what I commented on was the actual firebase code.. 🤦‍♂️